packages/mui-utils/src/getDisplayName/getDisplayName.test.tsx

```diff
@@ -41,13 +41,13 @@ describe('utils/getDisplayName.js', () => {
       ));
       NamedForwardRefComponent.displayName = 'Div';
 
-      const AnonymousMemoComponent = React.memo((props, ref) => <div {...props} ref={ref} />);
+      const AnonymousMemoComponent = React.memo((props) => <div {...props} />);
```

Comment:
Looks like removing the second `ref` argument in this memo components is the right call even if React 19 didn't remove it because these three components being modified are not using `React.forwardRef` anyway.

Reply:
Extracted to https://github.com/mui/material-ui/pull/43282